config.dev.toml

```diff
@@ -320,6 +320,15 @@ IndexType = "none"
         # Password is the password for a redis SingleFlight lock.
         # Env override: ATHENS_REDIS_PASSWORD
         Password = ""
+
+        [SingleFlight.Redis.LockConfig]
```

Comment:
We have config parsing tests can we update them?

Reply:
Updated the tests.